draft-ietf-mls-protocol.md

```diff
@@ -818,35 +818,21 @@ strings with length Hash.length to HPKE key pairs.
 Ciphersuites are represented with the CipherSuite type. HPKE public keys
 are opaque values in a format defined by the underlying Diffie-Hellman
 protocol (see the Ciphersuites section of the HPKE specification for more
-information):
+information).
 
 ~~~~~
-enum {
-    P256_SHA256_AES128GCM(0x0000),
-    X25519_SHA256_AES128GCM(0x0001),
-    (0xFFFF)
-} CipherSuite;
-
 opaque HPKEPublicKey<1..2^16-1>;
 ~~~~~
 
-### Curve25519, SHA-256, and AES-128-GCM
-
-This ciphersuite uses the following primitives:
-
-* Hash function: SHA-256
-* AEAD: AES-128-GCM
+The ciphersuites are defined in section {{MLS Ciphersuites}}.
```

Comment:
I think you need a bit more description here, just to say what a ciphersuite indicates, namely:

* Parameters for an instance of HPKE
* A KDF to be used 
* As a consequence, a Derive-Key-Pair function, as defined below.

Reply:
Also, does `{{MLS Ciphersuites}}` actually work?  Might need to be `{{mls-ciphersuites}}`.

Reply:
1) That's why I wanted to leave "HPKE" in the name of the ciphersuite
2) I guess that should then go in line 809
3) That's what we have in the "Diffie-Hellmann groups section"

Regarding the notation for hyperlinks, I'll take your word for it

Reply:
I fixed this in mine. The anchor is {#anchor} and then you link in text with {{anchor}}.